packages/apps-config/src/api/spec/interbtc.ts

```diff
@@ -63,10 +63,27 @@ export function getBalance (
   );
 }
 
+export function transferBalance (
+  account: string,
+  amount: number,
+  api: ApiPromise
+): SubmittableExtrinsic<'promise'> | undefined {
+  if (!api) {
+    return;
+  }
```

Comment:
Even though there is a check for the `api` not to be undefined before calling this function, unless this `if` is added the app crashes

Reply:
The signature for all derives are -

```js
// ReturnType is here for illustrative purposes
type ReturnType = string;

function myDerive (instanceId: string, api: ApiInterfaceRx): Observable<ReturnType> {
  return memo(instanceId, (): Observable<ReturnType> => {
    // we work with observables here
    return api.rpc.chain.getHeader().pipe(
      switchMap((header) => of(header.toString()))
    );    
  });
}
```